diff --git a/scripts/rabbitmq-server b/scripts/rabbitmq-server
index 71a6530..c86870f 100755
--- a/scripts/rabbitmq-server
+++ b/scripts/rabbitmq-server
@@ -21,7 +21,7 @@
 
 RABBITMQ_START_RABBIT=
 [ "x" = "x$RABBITMQ_ALLOW_INPUT" ] && RABBITMQ_START_RABBIT=" -noinput"
-[ "x" = "x$RABBITMQ_NODE_ONLY" ] && RABBITMQ_START_RABBIT="$RABBITMQ_START_RABBIT -s $RABBITMQ_BOOT_MODULE boot "
+[ "x" = "x$RABBITMQ_NODE_ONLY" ] && RABBITMQ_START_RABBIT="$RABBITMQ_START_RABBIT -s rabbit_clusterer boot -pa ${RABBITMQ_PLUGINS_DIR}/rabbitmq_clusterer-@RMQ_VERSION@.ez/rabbitmq_clusterer-@RMQ_VERSION@/ebin"
 
 case "$(uname -s)" in
   CYGWIN*) # we make no attempt to record the cygwin pid; rabbitmqctl wait
